src/librustc/lint/mod.rs

```diff
@@ -89,7 +89,6 @@ macro_rules! lint_initializer {
 /// Declare a static item of type `&'static Lint`.
 #[macro_export]
 macro_rules! declare_lint {
-    // FIXME(#14660): deduplicate
```

Comment:
I removed it because it seems impossible to do so now, you can't place a `pub` qualifier on a macro invocation (#22285).